main/test/ca/uwaterloo/flix/language/phase/TestParser.scala

```diff
@@ -317,4 +317,40 @@ class TestParser extends FunSuite with TestUtils {
     expectError[ParseError](result)
   }
 
+  test("ParseError.EOI.01") {
+    val input =
+      """
+        |def foo(): String = "
+        |""".stripMargin
```

Comment:
But will this not still have a \n? 

I think it should be simply 

`val input = "def foo(): String = \""`